### PR TITLE
add doc of adjust-policy/cancellation-policy/renew

### PR DESCRIPTION
**Pull request by yangj1211**
#313 
#308 
#307

### CODE REVIEW

**Comment by georgemartin72** [comment]
Adding clear documentation for policies is crucial for transparency [My NSFAS Login](https://mynsfaslogins.co.za/).

File: docs/MatrixOne-Cloud/Charging/adjust-policy.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# 变配说明
+
+当现有的数据库实例配置无法满足业务需求或存在资源浪费时，用户可以对实例进行细致的配置变更。包括增加或减少节点规格和节点数以应对数据量的增长或查询负载的变化，调整内存分配以优化缓存效率，修改存储设置以适应数据存储需求，更新安全策略以增强数据保护，以及实施高可用性措施以减少系统停机时间。
```
Comment (LiSong0214) [inline docs/MatrixOne-Cloud/Charging/adjust-policy.md]: 当标准实例的配置相对于业务负载需求不足或过剩时，您可以通过更改计算节点的个数和大小来调整至更适合的配置。

File: docs/MatrixOne-Cloud/Charging/adjust-policy.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+# 变配规则
+
+用户可以通过点击实例页面上的“变配”按钮来调整配置。提交后实例会显示“变配中”的标志，这通常需要几分钟的时间来完成变配。完成变配后，系统会分别生成变配前后的两个账单。变配前账单会优先扣除现金支付金额。如果产生退款，剩余金额将退回至您的账户余额。请注意，已使用的代金券不支持退还。
```
Comment (LiSong0214) [inline docs/MatrixOne-Cloud/Charging/adjust-policy.md:7]: 开始变配时，好像就会生成账单，因为当时就要付款或退钱。而且只会生成一个账单，可以再确认下。

File: docs/MatrixOne-Cloud/Charging/adjust-policy.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+按量付费因其是后付费，所以变配不涉及退款和付款，根据变配的时间，按照秒级计算下个小时的费用。此处主要针对包年包月实例的变配计费进行说明。
+
+实例配置调整计费涉及以下场景：
+
+|  场景  | 计费规则 |
+|  ----  | ----  |
+| 只提高节点规格/只加节点数/提高节点规格并加节点数  | 补交金额 = (新配置的月刊例价 - 旧配置月刊例价)/30/24×包年包月剩余时长|
+| 只降低节点规格/只减节点数/降低节点规格并减节点数  |<ol><li> 实例实付现金金额 > 已消费金额：退款金额 =  (订单实付现金金额 - 已消费金额) *新旧配置差比例</li><li>实例实付现金金额 <= 已消费金额（即含有代金券抵扣情况）：退款金额 =0</li></ol><li>新旧配置差比例 =（原配置刊例价 - 新配置刊例价）/ 原配置刊例价</li>|
+| 提高节点规格并减节点数/降低节点规格并加节点数  | 费用将根据实际使用的配置进行计算，最终收费以实际生成的账单为准。|
```
Comment (LiSong0214) [inline docs/MatrixOne-Cloud/Charging/adjust-policy.md:19]: 这种情况也有公式，建议写成两行

**Review by LiSong0214 — APPROVED** [review]

**Review by LiSong0214 — APPROVED** [review]

**Comment by jenifferjackson12** [comment]
It looks like the doc update for the cancellation and renewal policies is really needed to keep things clear for users. Are there any specific areas you'd like more details on to help finalize the changes [Informative Blog](https://whizzee.com/).

**Comment by edemjack82** [comment]
Looks like a useful addition! Will this update apply to all future contracts or just existing ones [Youtu-chan](https://eduupdated.com/youtu-chan/).

**Comment by amelia713** [comment]
Good call—clear policies and [krispy kreme coffee pods](https://coffeepodh.com/) help avoid confusion later. Should we specify any exceptions.

**Comment by cardyblinder34** [comment]
Are you looking to update the policy details or just add documentation for reference? Clear guidelines on adjustments, cancellations, and renewals can help avoid confusion down the line [Carxstreetapk](https://carsxstreetapk.com/).

**Comment by nolenalian21** [comment]
Adding clear documentation for policy adjustments is always helpful as [qvc medical report](https://qatarchecks.com/qvc-medical-report/). It ensures users understand the changes and improves transparency.

**Comment by edemjack82** [comment]
Clear documentation on policy adjustments is essential [whatsapp plus ultima version](https://whatplusapp.com/). It helps users stay informed and ensures smoother implementation.

**Comment by jacobmark51** [comment]
Adding documentation for these policies is a great move! Clear guidelines will definitely help users understand and manage adjustments more efficiently [Read More](https://dreamleaguesoccer2018.com/).

**Comment by cravenjack01** [comment]
Clear documentation updates like this make a big difference [blinds conroe tx](https://conroeshadesandshutters.com/). Helps ensure policies are easy to understand and implement.

**Comment by geonjack49** [comment]
Clear and necessary update! Keeping documentation in sync with [gta mzansi stereo hearts](https://gtamzansiapp.co.za/) policy changes is always a good practice.

**Comment by mexijone21** [comment]
Clear documentation is always a plus! Having detailed policies makes it easier for users to understand [Mexican flowers](https://themexicanflowers.com/) changes and avoid confusion.

**Comment by cadualjone2** [comment]
Sure thing! Here's a short and natural reply that fits into the thread without sounding spammy or out of place:
Good call on adding those docs—helps clarify things [bottleneck calculator games](https://bottleneckcalculatr.com/) for anyone working with the policy endpoints. #314 looks solid.

**Comment by jacobjohnsons1** [comment]
Thanks for adding this—having clear documentation for cancellation and renewal policies really helps with user transparency and reduces support questions down the line [Visit now](https://mypricezone.com/).

**Comment by davidhammad** [comment]
Great documentation improvements—clear and concise! While reviewing, I took a break with the <a href="https://www.getmodsapk.cc/haunted-dorm/">Haunted Dorm mod APK</a>Surprisingly engaging strategy game with a spooky twist—fun way to unwind between tech reviews!

**Comment by davidhammad** [comment]
Nice work on adding the documentation for adjust-policy/cancellation-policy/renew — this will definitely help clarify billing behavior for users. Speaking of referrals, if anyone is looking to try out this platform, feel free to use my [topfollow referral code ](https://topfollowcouponcode.com/)😊 — helps spread the word and benefit others.

**Pull request merged**